app/widgets/top-bar.less

```diff
@@ -33,4 +34,10 @@
       flex: 1;
     }
   }
+
+  .hamburger {
```

Comment:
i was going to leave the hamburger even on desktop, kind of weird but I'm seeing this done in a few places and i think it's not a bad idea

Reply:
I figured it was probably redundant if the left-nav is visible but it doesn't hurt.